src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1667,7 +1667,8 @@ private AuctionContext enrichWithHooksDebugInfo(AuctionContext context) {
 
         final ExtBidResponsePrebid updatedExtPrebid = ExtBidResponsePrebid.of(
                 extPrebid != null ? extPrebid.getAuctiontimestamp() : null,
-                extModules);
+                extModules,
+                extPrebid != null ? extPrebid.getTargeting() : Collections.emptyMap());
```

Comment:
This will always produce empty map, because on line 760 in `ExchangeService` you always set null for targetting:
`.prebid(ExtBidResponsePrebid.of(auctionTimestamp, null, null))`

Did I miss something?

Reply:
Yeah, that's correct. We don't want to modify the targeting key/values in the main auction in the prebid-server core.
This should be for modules so that any module can inject targeting key/values in the response hook